### DISCUSSION TITLE
Help: appending string to array value

### DISCUSSION THREAD

**Discussion by dmilan77** [mikefarah/yq#782] [Q&A] [answered]
Lets say I have an input array yml as `sample.yml`
```
a:
  - candy
  - apple
  - orange
  ```
 
I would like like to modify it by appending   a word `japan-` so that output looks like 
```
a:
  - japan-candy
  - japan-apple
  - japan-orange
  ```

I was trying this but all elements get updated with first element.
`yq eval '(.a.[] |  select(. == "*")) = "japan-"+.a.[]'  sample.yml`

**Comment by mikefarah** [comment, marked as answer]
Use the relative update `|=` , not at a machine with yq, but it will be something like:

yq eval '(.a.[]) |= "japan-"+.' sample.yml
